### ISSUE TITLE
Error loading external "[...].es6.js"

### ISSUE THREAD

**Issue by jasontheiler** [nuxt/framework#1755]
### Environment

- Operating System: `Linux`
- Node Version:     `v17.0.1`
- Nuxt Version:     `3.0.0-27268729.5b8e10f`
- Package Manager:  `pnpm@6.20.3`
- Bundler:          `Vite`
- User Config:      `buildModules`, `css`, `unocss`, `publicRuntimeConfig`
- Runtime Modules:  `-`
- Build Modules:    `nuxt-windicss@2.0.11`, `@unocss/nuxt@0.6.4`, `@vueuse/core/nuxt@6.8.0`

### Describe the bug

Whenever I try to use any of Motion One's functions (e.g. `animate`), Nuxt throws the error in the logs down below.

I suspect that it's caused by Motion One being an ESM only package and Nuxt 3 not being able to handle those, as described by Pooya in #245. But I'm not certain about this, which is why I created this issue. Maybe it's because of something different.

EDIT: I just realized that #245 was already fixed. So I really don't know what could cause this 😕

### Reproduction

On CodeSandbox and StackBlitz the same errors occur, when `animate` is used. If you comment it out or remove it, Nuxt 3 doesn't throw an error.

https://codesandbox.io/s/sad-shaw-89gcr?file=/app.vue:135-274

**OR**

https://stackblitz.com/edit/github-gmbgwg?devtoolsheight=33&file=app.vue

### Additional context

Using Motion One in a plain Vite project with Vue works just fine.

### Logs

```shell
(node:16264) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/home/jason/website/node_modules/.pnpm/tslib@2.3.1/node_modules/tslib/tslib.es6.js:24
export function __extends(d, b) {
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1025:15)
    at Module._compile (node:internal/modules/cjs/loader:1059:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1147:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:190:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
```

**Comment by danielroe** [comment]
@jasontheiler I haven't had a chance to look yet, but this may be helpful: http://v3.nuxtjs.org/concepts/esm

TL;DR: Nuxt supports esm, but often packages that use esm syntax are not doing so correctly.

**Comment by jasontheiler** [comment]
Oh, I'm sorry. I have totally overlooked that part of the docs 😭 

It appears to be an issue with the [`tslib`](https://github.com/microsoft/tslib) package that is used by Motion One internally. I was able to fix it by using the following config as described [here](http://v3.nuxtjs.org/concepts/esm#transpiling-libraries):
```ts
import { defineNuxtConfig } from "nuxt3";

// See: https://v3.nuxtjs.org/docs/directory-structure/nuxt.config
export default defineNuxtConfig({
  build: {
    transpile: ["tslib"],
  },
});
```

Thank you so much @danielroe!
